### PR TITLE
Added thumbs.TemplateView

### PR DESCRIPTION
**Pull request by doug-martin**

### CODE REVIEW

File: thumbs.js [hunk 1]
```diff
@@ -244,6 +243,17 @@
 
     };
 
+    thumbs.templater = (function _templater(templater) {
```
Comment (nicknisi) [inline thumbs.js]: What is the templater argument? Is it used?

File: thumbs.js [hunk 2]
```diff
@@ -218,8 +218,7 @@
     var Formatter = {
```
Comment (nicknisi) [inline thumbs.js:217]: You never see other plugins mixing their docs with code. I really think this makes things messy and confusing. 

This plugin is too simple not to just make this available in the readme. Yes I know you can run your fancy generator, but why muck up the code when you can write simple markdown?

File: thumbs.js [hunk 3]
```diff
@@ -101,13 +101,13 @@
     Router = thumbs.Router = Router.extend(_super);
 
     var EventDelegator = {
-        render: function render () {
+        render: function render() {
             this._super('render', arguments);
```
Comment (nicknisi) [inline thumbs.js:103]: Run lint. This line won't pass

Comment (doug-martin) [inline thumbs.js:103]: It did pass :)

**Comment by nicknisi** [comment]
Looks good

**Pull request merged**